comms/peer/src/Peer.ts

```diff
@@ -143,6 +143,10 @@ export class Peer implements IPeer {
 
       this.pingTimeoutId = schedulePing();
     }
+    
+    this.stats = new GlobalStats(this.config.statsUpdateInterval ?? 1000);
```

Comment:
Super nit: maybe you could make this a constant somewhere, so it's easier to find/understand